crates/core/src/test.rs

```diff
@@ -12725,3 +12748,51 @@ fn go_package_type() {
     })
     .unwrap();
 }
+
+#[test]
+fn call_php() {
+    run_test_no_match({
+        TestArg {
+            pattern: r#"
+                |language php
+                |
+                |`TEST`
+                |"#
+            .trim_margin()
+            .unwrap(),
+            source: r#"
+                |echo "hello world"
+                |"#
+            .trim_margin()
+            .unwrap(),
+        }
+    })
+    .unwrap();
+}
+
+
+#[test]
+fn call_php_2() {
+    run_test_expected({
+        TestArgExpected {
+            pattern: r#"
+                |language php
+                |
+                |`echo $str` => `echo "modified"`
```

Comment:
`$str` is a php variable so we grit metavariables should be something different. I recommend `#str`

Reply:
# is used for comments in php. So I switched to using ^ as a prefix instead.